app/src/main/res/layout/fragment_media_detail.xml

```diff
@@ -40,257 +39,273 @@
 
             <!-- Placeholder. Height gets set at runtime based on container size; the initial value is a hack to keep
                  the detail info offscreen until it's placed properly. May be a better way to do this. -->
-            <com.facebook.drawee.view.SimpleDraweeView
-                android:id="@+id/mediaDetailImageView"
+
+            <LinearLayout
                 android:layout_width="match_parent"
                 android:layout_height="@dimen/dimen_250"
-                app:actualImageScaleType="none" />
+                android:orientation="vertical"
+                android:background="@android:color/transparent"
```

Comment:
transparent is the default color(unless there has been some theme magic), can be omitted

Reply:
@kbhardwaj123 drawing your attention to this comment as the PR is quite difficult to view in the main `conversation`